conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -44,23 +38,17 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
-    - cmd: rmdir C:\cygwin /s /q
```

Comment:
Just a question: if this is moved to `conda-forge-build-setup`, then this makes `conda-forge-build-setup` appveyor specific. Is that intended or should that package be kept as a "install it any you've all the settings and packages which are needed to start conda-forge package builds"-package?